src/DynamoApplications/StartupUtils.cs

```diff
@@ -53,20 +50,19 @@ internal class CMDLineOptions
         public string UserDataFolder { get; set; } = String.Empty;
         [Option("CommonData", Required = false, HelpText = "Specify common data folder to be used by PathResolver with CLI.")]
         public string CommonDataFolder { get; set; } = String.Empty;
-        [Option("HostName", Required = false, HelpText = "Identify Dynamo variation associated with host.")]
+        [Option('t', "HostName", Required = false, HelpText = "Identify Dynamo variation associated with host.")]
```

Comment:
Same comment here as above.

Reply:
I'm not so sure about it, as far as I know, the recommendation is using single-character flags but I could try your suggestion.

Reply:
Solved.